floor_generator/pubspec.yaml

```diff
@@ -17,8 +17,15 @@ dependencies:
   source_gen: ^0.9.4+7
   build_config: ^0.4.1+1
   collection: ^1.14.11
+  strings: ^0.1.2
   floor_annotation:
     path: ../floor_annotation/
+  sqlparser: # should be ^0.11.0
+    git:
+      url: https://github.com/simolus3/moor.git
+      ref: c2ce934
+      path: sqlparser
+
```

Comment:
FYI: I just released version `0.10.1` which contains the commit referenced here. There weren't any breaking changes, which is why it's not `0.11.0`

Reply:
ah well, this was just to remind me to use the next version, not 0.11.0 explicitly. Thank you for the notification!